pkg/agent/utils.go

```diff
@@ -393,19 +411,23 @@ func getDynamicKubeletConfigFileContent(kc map[string]string) string {
 		StaticPodPath: kc["--pod-manifest-path"],
 		Authorization: KubeletAuthorization{
 			Mode: KubeletAuthorizationMode(kc["--authorization-mode"]),
+			Webhook: KubeletWebhookAuthorization{
+				CacheAuthorizedTTL:   Duration(KubeletCacheAuthorizedTTL),
+				CacheUnauthorizedTTL: Duration(KubeletCacheUnauthorizedTTL),
+			},
```

Comment:
Do we get any benefit with this explicit setting?

Reply:
removed